SocialCareCaseViewerApi/V1/Infrastructure/MashReferral_2.cs

```diff
@@ -0,0 +1,74 @@
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+#nullable enable
+namespace SocialCareCaseViewerApi.V1.Infrastructure
+{
+
+    [Table("ref_mash_referrals", Schema = "dbo")]
+    public class MashReferral_2 : IAuditEntity
+    {
+
+
+
+        [Column("id")]
+        [MaxLength(16)]
+        [Key]
+        public long Id { get; set; }
+        [Column("referrer")]
+        public string Referrer { get; set; } = null!;
+        [Column("requested_support")]
+        public string RequestedSupport { get; set; } = null!;
+        [ForeignKey("worker_id")]
+        public Worker? AssignedTo { get; set; }
```

Comment:
Remove this until we add the ability to link a worker, remove from domain and response classes too (also entity factory and response factory)